tests/sentry/tasks/test_auto_resolve_issues.py

```diff
@@ -75,3 +79,82 @@ def test_simple(self, mock_kick_off_status_syncs, mock_backend, mock_record):
             issue_type="error",
             issue_category="error",
         )
+
+    @patch("sentry.tasks.auto_ongoing_issues.backend")
+    @patch("sentry.tasks.auto_resolve_issues.kick_off_status_syncs")
+    def test_legacy_performance(self, mock_kick_off_status_syncs, mock_backend):
+        project = self.create_project()
+
+        current_ts = int(time()) - 1
+
+        project.update_option("sentry:resolve_age", 1)
+
+        group = self.create_group(
+            project=project,
+            status=GroupStatus.UNRESOLVED,
+            last_seen=timezone.now() - timedelta(days=1),
+            type=PerformanceSlowDBQueryGroupType.type_id,  # Test that auto_resolve is enabled for legacy performance issues
+        )
+
+        mock_backend.get_size.return_value = 0
+
+        with self.tasks():
+            schedule_auto_resolution()
+
+        assert Group.objects.get(id=group.id).status == GroupStatus.RESOLVED
+
+        mock_kick_off_status_syncs.apply_async.assert_called_once_with(
+            kwargs={"project_id": group.project_id, "group_id": group.id}
+        )
+
+        assert project.get_option("sentry:_last_auto_resolve") > current_ts
+
+    @patch("sentry.tasks.auto_ongoing_issues.backend")
+    @patch("sentry.tasks.auto_resolve_issues.kick_off_status_syncs")
+    def test_SD_performance(self, mock_kick_off_status_syncs, mock_backend):
+        project = self.create_project()
+
+        current_ts = int(time()) - 1
+
+        project.update_option("sentry:resolve_age", 1)
+
+        group = self.create_group(
+            project=project,
+            status=GroupStatus.UNRESOLVED,
+            last_seen=timezone.now() - timedelta(days=1),
+            type=PerformanceDurationRegressionGroupType.type_id,  # Test that auto_resolve is disabled for SD
+        )
+
+        mock_backend.get_size.return_value = 0
+
+        with self.tasks():
+            schedule_auto_resolution()
+
+        assert Group.objects.get(id=group.id).status == GroupStatus.RESOLVED
+
+        mock_kick_off_status_syncs.apply_async.assert_called_once_with(
+            kwargs={"project_id": group.project_id, "group_id": group.id}
+        )
+
+        assert project.get_option("sentry:_last_auto_resolve") > current_ts
+
+    @patch("sentry.tasks.auto_ongoing_issues.backend")
+    def test_SD_performance_ff_enabled(self, mock_backend):
+        with self.feature("organizations:issue-platform-api-crons-sd"):
```

Comment:
test_legacy_performance should also set the feature flag, right? Also, we generally prefer the `@with_feature` decorator

Reply:
I went ahead and renamed legacy to single_event and sd to aggregate